src/consensus/ledger_enclave_types.h

```diff
@@ -6,7 +6,7 @@
 
 namespace consensus
 {
-  using Index = uint64_t;
+  using Index = int64_t;
```

Comment:
Why is that signed now?

Reply:
Good point. This wasn't meant to change so I've reverted this and the snapshot now use the unsigned `consensus::Index` instead of the signed `kv::Version`.